packages/feedback/src/util/prepareFeedbackEvent.ts

```diff
@@ -17,20 +15,22 @@ export async function prepareFeedbackEvent({
   scope,
   event,
 }: PrepareFeedbackEventParams): Promise<FeedbackEvent | null> {
-  const eventHint = { integrations: undefined };
+  const eventHint = {};
   if (client.emit) {
     client.emit('preprocessEvent', event, eventHint);
   }
 
   const preparedEvent = (await prepareEvent(
```

Comment:
Just to be clear, since we talked about this yesterday: If this goes through `prepareEvent` it goes through all event processors, as that applies event processors on the event!

Reply:
👍 I think this is what we want, for it to go through event processors but not beforeSend